gateway-applications/src/main/resources/applications/webshell/app/libs/bower/jquery/js/jquery-3.5.1.min.js

```diff
@@ -0,0 +1,2 @@
+/*! jQuery v3.5.1 | (c) JS Foundation and other contributors | jquery.org/license */
```

Comment:
Do we need this copied in? Can we follow the pattern of other applications using npm to at least keep it reasonable to upgrade in the future?